src/index.js

```diff
@@ -47,6 +47,10 @@ function init(tenantId, opts = {}) {
     ] = `https://${store.domain}`;
   }
 
+  if (opts.originHeader) {
```

Comment:
Maybe it would be better to call this options `originHeaderUrl` to be more explicit?